code/data-processing/assemble-historical-jhu.R

```diff
@@ -2,14 +2,15 @@ library(readr)
 library(purrr)
 library(tibble)
 library(here)
+library(covidData)
 
 # after this your working directory should be covidData
 setwd(here())
 
```

Comment:
sourcing the function from covidData that we need here (since I took out `library(covidData)` above)
```suggestion
source("R/calc_jhu_global_cum.R")
```